src/Orleans.Transactions.Azure/Hosting/SiloBuilderExtensions.cs

```diff
@@ -1,7 +1,6 @@
-﻿using Microsoft.Extensions.DependencyInjection;
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
 using Orleans.Hosting;
-using Orleans.Transactions.Abstractions;
-using System.Threading.Tasks;
 
 namespace Orleans.Transactions.Azure
```

Comment:
I wonder if we should put these extension methods in a more common namespace to improve discovery. No strong opinion.